tests/test_linear_ancilla_composite.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2018 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import unittest
+import collections
+
+import numpy as np
+
+import networkx as nx
+
+import dimod
+from dimod import TrackingComposite, StructureComposite
+
+from dwave.system.testing import MockDWaveSampler
+from dwave.system import LinearAncillaComposite
```

Comment:
```suggestion
import collections
import unittest

import dimod
import networkx as nx
import numpy as np

from dimod import TrackingComposite, StructureComposite

from dwave.system.testing import MockDWaveSampler
from dwave.system import LinearAncillaComposite
```

I would suggest that `dimod.TrackingComposite` and `dimod.StructureComposite` will be easier for readers to follow in the body of the code. Rather than importing them directly into the namespace.